src/dragonball/src/dbs_virtio_devices/src/balloon.rs

```diff
@@ -675,11 +675,14 @@ where
             error!("Failed to write config space");
             return Err(ConfigError::InvalidOffset(offset));
         };
-        let Some(dst) = start.checked_add(data.len())
-            .and_then(|end| config_slice.get_mut(start..end)) else
-        {
+        let Some(dst) = start
```

Comment:
why change here？

Reply:
it's automated done by the newest rust version cargo fmt

Reply:
Can you split it into two patches

Reply:
done